Wurst Client for MC 1.9.X/src/tk/wurst_client/mods/MassTell.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright © 2014 - 2016 | Wurst-Imperium | All rights reserved.
+ * 
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+ /*
+ * Made by dGRAMOP for WURST!!!
```

Comment:
You think so? Let's wait for @Alexander01998 to answer this.

@Alexander01998 wants to add credits like [this](https://github.com/Wurst-Imperium/wurst-client.tk/issues/101).